src/gui/wizard/owncloudwizard.h

```diff
@@ -87,6 +88,7 @@ class OwncloudWizard : public QWizard
     QSslKey _clientSslKey; // key extracted from pkcs12
     QSslCertificate _clientSslCertificate; // cert extracted from pkcs12
     QList<QSslCertificate> _clientSslCaCertificates;
+    bool useFlow2() { return _useFlow2; };
```

Comment:
Please move this above with the other methods and implement within the `.cpp` file